### DISCUSSION TITLE
totally remove date from pages

### DISCUSSION THREAD

**Discussion by ilyessegit** [chipzoller/hugo-clarity#441] [Q&A] [answered]
I hope you all have a great day in this hot summer !

Im using an HUGO [docker image 1](https://hub.docker.com/r/klakegg/hugo) with the [Clarity 2](https://github.com/chipzoller/hugo-clarity) theme for my project and everything works perfectly fine !

Except that i have pages with RSS feed, and my issue is that by default there is a wrong date that i can’t remove.

<img width="938" alt="image" src="https://github.com/chipzoller/hugo-clarity/assets/38523606/fda6f167-4722-4c31-abd4-8bd0334fd45e">

I don’t know if there is a way, in the config.toml or directly in the page to remove the date ? And my layouts folder is empty so i can't modify it.

Thanks :slight_smile:

**Comment by ilyessegit** [comment]
Well i saw that i can add "showdate = false" into the parameters file.

The thing is that i need this only for one page.

Is there a way to condition to do that for only a specific page ?

Thanks :)

**Reply by onweru** [reply, marked as answer]
Not yet but we can add that when we have the run-way for that. Or you can submit a PR for the same
